v_3_1_0/master_template.go

```diff
@@ -1768,6 +1768,211 @@ write_files:
         omitStages:
           - "RequestReceived"
 
+- path: /etc/kubernetes/manifests/k8s-api-server.yml
+  owner: root
+  permissions: 0644
+  content: |
+    apiVersion: v1
+    kind: Pod
+    metadata:
+      name: k8s-api-server
+      namespace: kube-system
+    spec:
+      hostNetwork: true
+      containers:
+      - name: k8s-api-server
+        image: quay.io/giantswarm/hyperkube:v1.9.2
+        env:
+        - name: HOST_IP
```

Comment:
Difference from systemd unit, where we used `/etc/network-environment` for getting env variable with host ip address.

Reply:
ack.